apps/editors/views.py

```diff
@@ -183,25 +191,30 @@ def home(request):
         durations=durations,
         reviews_max_display=reviews_max_display,
         motd_editable=motd_editable,
-        queue_counts_total=queue_counts(admin_reviewer=True),
+        queue_counts_total=queue_counts(admin_reviewer=True,
+                                        limited_reviewer=limited_reviewer),
         unlisted_queue_counts_total=queue_counts(admin_reviewer=True,
-                                                 unlisted=True),
+                                                 unlisted=True,
+                                                 limited_reviewer=limited_reviewer),
```

Comment:
This raises a flake8 error:
./apps/editors/views.py:198:80: E501 line too long (84 > 79 characters)

How to fix this?